releasenotes/notes/fix-broken-a5-import-a2358410cf4a37da.yaml

```
@@ -0,0 +1,16 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Agent 5 configuration import was missing critical parameters.
+    The configuration was reset to default settings in the middle
+    of the migration process. It was leading to many unimported
+    parameters including very important ones like ``api_key``.
+    Some Docker & Kubernetes configuration settings were also
+    imported in wrongly located files.
```

nit: for conciseness:
```suggestion
    Fix bugs introduced in 7.20.0/6.20.0 in the Agent 5 configuration import command:
    the command would not import some Agent config settings, including ``api_key``,
    and would write some Docker & Kubernetes config settings to wrongly-located files.
```